src/Common/src/System/Data/DataTableExtensions.cs

```diff
@@ -12,7 +12,7 @@ namespace System.Data
     /// <summary>
```

Comment:
We still build a `netstandard` implementation here https://github.com/dotnet/corefx/blob/master/src/System.Data.DataSetExtensions/src/System.Data.DataSetExtensions.csproj.

This implementation is used when the package is installed on netstandard2.0 frameworks that aren't netcoreapp3.0 or higher (eg: netcoreapp2.x, uap10.0.16299, xamarin frameworks).  We could decide to stop building that in master and just copy it from servicing but it'd mean that you need to make changes in servicing branches rather than master if you wanted to change that implementation.

Note that either way I will ensure that the change does not leave behind duplicate files.

Reply:
Ok, I didn't know that. But I believe we don't build System.Data.Common as a package from master anymore. Does that make stopping building System.Data.DataSetExtensions from master make more sense?

Reply:
The System.Data.Common package was eliminated for an different reason, the surface area was absorbed into netstandard2.0, as well as the facade.  Because of this netstandard2.0 supporting frameworks must have the dll inbox, and the netstandard targeting pack included a facade to unify the dll to types in netstandard.dll.  As such, shipping a package would be pretty pointless since all recent frameworks had to have the DLL inbox.

Contrast that to DataSetExtensions: we aren't getting rid of this DLL.  We still need reference assembly for netstandard, we need facade for netcoreapp3.0, and implementations for other frameworks.

If you eliminated the package you'd still need to determine how to deliver the DataSetExtensions facade: put it inbox in .NETCore 3.0?  If you're willing to do this, that'd work.

If not, then wait for next release to eliminate the package in master.  That way you can use the 3.0 servicing branch to service the downlevel implementations while still producing the facade needed for 3.0.

Reply:
Makes sense. Thanks for the detailed explanation.

Reply:
> put it inbox in .NETCore 3.0? If you're willing to do this, that'd work.

Would there be any downside to this? It would mean you get  compatibility with libraries built against desktop without any additional package, right?

Reply:
Initially I was not thrilled about adding another DLL inbox, but given that it doesn't add any dependencies and we could think of it like the desktop compat shims I feel less strongly.  It also has the benefit of avoiding a compiler error when folks reference the old package and target netcoreapp3.0.
@danmosemsft @stephentoub any objections?

Reply:
Size is not an issue here right?

Reply:
Signed facades with a similar number of type forwards are 14KB so that's our upper bound.

Reply:
I have no objection.